std/math.d

```diff
@@ -1592,21 +1585,10 @@ real exp(real x) @trusted pure nothrow @nogc
         enum real C2 = 1.428606820309417232121458176568075500134E-6L;
 
         // Overflow and Underflow limits.
-        static if (real.mant_dig == 64) // 80-bit reals
```

Comment:
Don't you think performing these 2 changes in a proper commit would be just as much work as this revert+description?